### PR TITLE
Replacing Big with GN in withdraw and send modals

### PR DESCRIPTION
**Pull request by IanWoodard**
This marks the first two usages of the proposed GN. To accommodate what I needed, I added two new methods to check if a number is zero or positive. Additionally, this is the first usage, so there may be some pain points, such as having to deal with undefined, that we may want to refine.

### CODE REVIEW

File: earn/src/components/portfolio/modal/SendCryptoModal.tsx [hunk 1]
```diff
@@ -207,8 +207,12 @@ export default function SendCryptoModal(props: SendCryptoModalProps) {
     setSelectedOption(defaultOption);
   }, [defaultOption]);
 
-  const bigSendAmount = new Big(sendAmountInputValue || 0).mul(String1E(selectedOption.decimals));
-  const bigSendBalance = new Big(depositBalance?.formatted ?? 0).mul(String1E(selectedOption.decimals));
+  const gnSendAmount = sendAmountInputValue
+    ? GN.fromDecimalString(sendAmountInputValue, selectedOption.decimals)
+    : GN.fromDecimalString('0', selectedOption.decimals);
```
Comment (haydenshively) [inline earn/src/components/portfolio/modal/SendCryptoModal.tsx:212]:
```suggestion
  const gnSendAmount = GN.fromDecimalString(sendAmountInputValue ? sendAmountInputValue : '0', selectedOption.decimals)
```

File: earn/src/components/portfolio/modal/WithdrawModal.tsx [hunk 2]
```diff
@@ -243,9 +244,13 @@ export default function WithdrawModal(props: WithdrawModalProps) {
     };
   }, [refetchMaxWithdraw, refetchMaxRedeem, isOpen]);
 
-  const bigWithdrawAmount = inputValue ? new Big(inputValue).mul(10 ** selectedOption.decimals) : new Big(0);
-  const bigMaxWithdraw: Big = maxWithdraw ? toBig(maxWithdraw) : new Big(0);
-  const bigMaxRedeem: Big = maxRedeem ? toBig(maxRedeem) : new Big(0);
+  const gnWithdrawAmount = inputValue
+    ? GN.fromDecimalString(inputValue, selectedOption.decimals)
+    : GN.fromDecimalString('0', selectedOption.decimals);
```
Comment (haydenshively) [inline earn/src/components/portfolio/modal/WithdrawModal.tsx:249]:
```suggestion
  const gnWithdrawAmount = GN.fromDecimalString(inputValue ? inputValue : '0', selectedOption.decimals)
```

File: earn/src/components/portfolio/modal/WithdrawModal.tsx [hunk 3]
```diff
@@ -243,9 +244,13 @@ export default function WithdrawModal(props: WithdrawModalProps) {
...
+  const gnMaxWithdraw =
+    GN.fromBigNumber(maxWithdraw, selectedOption.decimals) ?? GN.fromDecimalString('0', selectedOption.decimals);
```
Comment (haydenshively) [inline earn/src/components/portfolio/modal/WithdrawModal.tsx:251]:
```suggestion
  const gnMaxWithdraw =
    GN.fromBigNumber(maxWithdraw ?? BigNumber.from('0'), selectedOption.decimals)
```

File: earn/src/components/portfolio/modal/WithdrawModal.tsx [hunk 4]
```diff
@@ -243,9 +244,13 @@ export default function WithdrawModal(props: WithdrawModalProps) {
...
+  const gnMaxRedeem =
+    GN.fromBigNumber(maxRedeem, selectedOption.decimals) ?? GN.fromDecimalString('0', selectedOption.decimals);
```
Comment (haydenshively) [inline earn/src/components/portfolio/modal/WithdrawModal.tsx:253]:
```suggestion
  const gnMaxRedeem =
    GN.fromBigNumber(maxRedeem ?? BigNumber.from('0'), selectedOption.decimals);
```

**Review by haydenshively — COMMENTED** [review]
I know you'll hit most of these on your own, but figured it wouldn't hurt to comment.

File: earn/src/components/portfolio/modal/WithdrawModal.tsx (hunk 2 above)
Comment (IanWoodard) [reply earn/src/components/portfolio/modal/WithdrawModal.tsx]: Went with a slightly more compact version

**Review by haydenshively — APPROVED** [review]
LGTM

**Pull request merged**